packages/hardhat/hardhat.config.ts

```diff
@@ -55,7 +55,7 @@ const config: HardhatUserConfig = {
       },
     },
     mainnet: {
-      url: `https://eth-mainnet.alchemyapi.io/v2/${providerApiKey}`,
+      url: "https://eth.drpc.org",
```

Comment:
Was planning to add a comment referencing issue #1126 above this line and also commented Alchemy URL but then thought since this is just hot-fix and when https://github.com/wighawag/hardhat-deploy/issues/571 is solved we can revert back. 

But happy to add a comment 🙌